frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -39,11 +41,16 @@ const ItemRow = ({item}) => {
     viewAPIExplorerMenuItem(item.kind, apiExplorerLink),
     ...(item.kind === 'OAuth') ? oauthMenuItems : [],
   ];
+  const itemKey = getDefinitionKey(item.model, allDefinitions);
+  const itemDefinition: SwaggerDefinition = _.get(allDefinitions, itemKey) || {};
```

Comment:
This should use `getResourceDescription`

https://github.com/openshift/console/blob/master/frontend/public/module/k8s/swagger.ts#L51-L58